### PR TITLE
[zh-cn] Translate mutating-webhook-configuration-v1.md

### PR DESCRIPTION
**Pull request by donghui12**
Trablate content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md
https://github.com/kubernetes/website/issues/34733

### CODE REVIEW

**Comment by tengqm** [comment]
/retitle [zh-cn] Translate mutating-webhook-configuration-v1.md

File: content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md [hunk 1]
```diff
@@ -0,0 +1,1113 @@
...
+      <!-- 
+      - **webhooks.clientConfig.caBundle** ([]byte)
+
+        `caBundle` is a PEM encoded CA bundle which will be used to validate the webhook's server certificate. If unspecified, system trust roots on the apiserver are used. 
+      -->
+
+        - **webhooks.clientConfig.caBundle** ([]byte)
+
+          `caBundle` 是一个 PEM 编码的 CA 包，将用于验证 Webhook 的服务证书。如果未指定，则使用 apiserver 上的系统信任根。
+
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md:107]: 上面几行为什么多缩进两个空格？

File: content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md [hunk 2]
```diff
@@ -0,0 +1,1113 @@
...
+        `service` is a reference to the service for this webhook. Either `service` or `url` must be specified.
+        
+        If the webhook is running within the cluster, then you should use `service`.
+
+        <a name="ServiceReference"></a>
+        *ServiceReference holds a reference to Service.legacy.k8s.io* 
+      -->
+
+        - **webhooks.clientConfig.service** (ServiceReference)
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md:119]: 下面这个条目也多缩进了两个空格！

File: content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md [hunk 3]
```diff
@@ -0,0 +1,1113 @@
...
+          <a name="ServiceReference"></a>
+          **ServiceReference 持有对 Service.legacy.k8s.io 的引用**
+
+          <!-- 
+          - **webhooks.clientConfig.service.name** (string), required
+
+            `name` is the name of the service. Required 
+          -->
+
+            - **webhooks.clientConfig.service.name** (string), 必需
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md:134]: 请先把缩进修正为英文版格式再继续 review。

File: content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md [hunk 4]
```diff
@@ -0,0 +1,1124 @@
...
+      对 apps/v1beta1 或 extensions/v1beta1 的请求不会被发送到 Webhook。
+
+    - Equivalent: 如果针对的资源包含在 “rules” 中，即使请求是通过另一个 API 组或版本提交，也会匹配。
+      例如，如果可以通过 apps/v1、apps/v1beta1 和 extensions/v1beta1 修改 deployments 资源，
+      并且 “rules” 仅包含 `apiGroups:["apps"]、apiVersions:["v1"]、resources:["deployments "]`，
+      对 apps/v1beta1 或 extensions/v1beta1 的请求将被转换为 apps/v1 并发送到 Webhook。
+
+    默认为 “Equivalent”。
+
+    - **webhooks.namespaceSelector** (<a href="{{< ref "../common-definitions/label-selector#LabelSelector" >}}">LabelSelector</a>)
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md:264]: 264 的缩进怎么和248 行不同？
请彻查缩进问题。

**Comment by tengqm** [comment]
preview: https://deploy-preview-34991--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1/

**Comment by tengqm** [comment]
image

**Comment by tengqm** [comment]
image

**Comment by tengqm** [comment]
image

File: content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md [hunk 5]
```diff
@@ -0,0 +1,1133 @@
...
+          "operator": "In",
+          "values": [
+            "prod",
+            "staging"
+          ]
+        }
+      ]
+    }
+    ```
+    <!--
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md:313]: 313 前面加空行

File: content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md [hunk 6]
```diff
@@ -0,0 +1,1133 @@
...
+        }
+      ]
+    }
+    ```
+    <!-- 
+    See https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/ for more examples of label selectors.
+
+    Default to the empty LabelSelector, which matches everything.  
+    -->
+    有关标签选择算符的更多示例，请参阅
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md:318]: 318 前面加空行。

File: content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md [hunk 7]
```diff
@@ -0,0 +1,1133 @@
...
+    <!-- 
+    reinvocationPolicy indicates whether this webhook should be called multiple times as part of a single admission evaluation. Allowed values are "Never" and "IfNeeded".
+
+        Never: the webhook will not be called more than once in a single admission evaluation.
+    
+    IfNeeded: the webhook will be called at least one additional time as part of the admission evaluation if the object being admitted is modified by other admission plugins after the initial webhook call. Webhooks that specify this option *must* be idempotent, able to process objects they previously admitted. Note: * the number of additional invocations is not guaranteed to be exactly one. * if additional invocations result in further modifications to the object, webhooks are not guaranteed to be invoked again. * webhooks that use this option may be reordered to minimize the number of additional invocations. * to validate an object after all mutations are guaranteed complete, use a validating admission webhook instead.
+    
+    Defaults to "Never". 
+    -->
+    reinvocationPolicy 表示这个 webhook 是否可以被多次调用，作为一次准入评估的一部分。可取值有 “Never” 和 “IfNeeded”。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md:350]: 350 前面加空行。

File: content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md [hunk 8]
```diff
@@ -0,0 +1,1133 @@
...
+
+    - Never: 在一次录取评估中，webhook 被调用的次数不会超过一次。
+    - IfNeeded：如果被录取的对象在被最初的 Webhook 调用后又被其他录取插件修改，
+      那么该 webhook 将至少被额外调用一次作为录取评估的一部分。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1.md:354]: webhook -> Webhook，请彻查

**Comment by tengqm** [comment]
@donghui12 Please use the https://deploy-preview-34991--kubernetes-io-main-staging.netlify.app/zh-cn/docs/reference/kubernetes-api/extend-resources/mutating-webhook-configuration-v1/ URL for preview rather than other software.

**Comment by donghui12** [comment]
image
I don't know it about this。

**Comment by tengqm** [comment]
> I don't know it about this。

After adding empty lines, the problem has been fixed.

/lgtm
/approve

**Pull request merged**